spartan/ethereum-testnet/reference/network_params.yaml

```diff
@@ -0,0 +1,198 @@
+#
```

Comment:
I have dropped in a reference from the ethereum-package repo here to make referencing it easier when adding new configurations